### PR TITLE
Codecoverage: filter test binaries and force DataX.Config.dll to build pdbs

### PR DESCRIPTION
**Pull request by carlbrochu**
The Config.dll wouln't be included in the instrumented binaries until I applied a DebugType parameter in the project to force building a pdb.  This doesn't have any other effect.

I also added a filter for the test binaries with a plural filename.

### CODE REVIEW

File: Services/DataX.Config/DataX.Config.LivyClient.Test/JobOperationTest.cs [hunk 1]
```diff
@@ -36,6 +36,51 @@ public async Task TestStartJob()
             Assert.AreEqual(expected: JobState.Running, actual: refreshJobInfo.JobState);
         }
 
+        [TestMethod]
```
Comment (kjcho-msft) [inline Services/DataX.Config/DataX.Config.LivyClient.Test/JobOperationTest.cs]: I think I saw these changes in your previous PR?

File: Services/codecoverage.runsettings [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+  <DataCollectionRunSettings>
+    <DataCollectors>
+      <DataCollector friendlyName="Code Coverage"
+      uri="datacollector://Microsoft/CodeCoverage/2.0" assemblyQualifiedName="Microsoft.VisualStudio.Coverage.DynamicCoverageDataCollector,
+ Microsoft.VisualStudio.TraceCollector, Culture=neutral">
+        <Configuration>
+          <CodeCoverage>
+            <ModulePaths>
+              <Include>
+                <ModulePath>.*\DataX.*.dll</ModulePath>
```
Comment (kjcho-msft) [inline Services/codecoverage.runsettings]: Should we include Flow.*.exe as well? e.g. Flow.InteractiveQueryService.exe

File: Services/codecoverage.runsettings [hunk 3]
```diff
@@ -0,0 +1,25 @@
...
+              </Include>
+              <Exclude>
+                <ModulePath>.*tests.dll</ModulePath>
+                <ModulePath>.*test.dll</ModulePath>
```
Comment (kjcho-msft) [inline Services/codecoverage.runsettings]: Do we need both *tests.dll and *test.dll?

Comment (carlbrochu) [reply Services/codecoverage.runsettings]: Yes we have both being built today.

Comment (carlbrochu) [reply Services/codecoverage.runsettings]: And didn't want to do that many renames in this pr

**Pull request closed without merging**